README.rst

```diff
@@ -111,3 +111,12 @@ See `COPYING <COPYING>`_ to see the full text.
    :target: https://app.shippable.com/projects/573f79d02a8192902e20e34b
 .. |Chat badge| image:: https://img.shields.io/badge/chat-IRC-brightgreen.svg
    :target: https://docs.ansible.com/ansible/latest/community/communication.html
+.. |Code Of Conduct| image:: https://img.shields.io/badge/Code%20of%20Conduct-Ansible-silver.svg
+   :target: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
+   :alt: Ansible Code of Conduct
+.. |Mailing Lists| image:: https://img.shields.io/badge/Mailing%20lists-Ansible-orange.svg
```

Comment:
Make the badge label lowercase since it's not a proper noun and that will match the other badges:

```suggestion
.. |Mailing Lists| image:: https://img.shields.io/badge/mailing%20lists-Ansible-orange.svg
```

Reply:
This matches recent changes made to molecule and ansible-lint.

cc @gundalow what do you think?

Reply:
OTOH, you're probably right about style consistency. Applying the suggestions now...